src/components/Facts/Facts.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  data: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
+function buildFactElements(facts) {
+  const factElements = [];
+  let factElementGroup = [];
+
+  facts.forEach((fact, index) => {
+    const element = (
+      <p key={index.toString()} className="f5 f4-ns lh-copy mt0 text-our-gray">
+        {fact}
+      </p>
+    );
+
+    factElementGroup.push(element);
+
+    if ((index + 1) % 3 === 0) {
+      factElements.push(<div key={index.toString()} className="fl w-third pa2">{factElementGroup}</div>);
```

Comment:
How responsive does this approach end up being? What happens when we squeeze the page thinner?

Reply:
It looks the same. Which, to be clear, is not great. This is a section that might be worth reworking a bit if we think optimizing views for mobile is a priority.

Honestly, this code that dynamically generates the elements is a tad bit unnecessary. It was mostly a chance for me to limber up on some React techniques and add a small amount of flexibility into the code. However, it's not _that_ flexible.

Reply:
I feel you. It was identical locally which is what I want for now because I just wanted to get on the stack without regression. When we overhaul this UI in the nearish future, this will almost certainly go away.